passwd.c

```diff
@@ -93,5 +106,20 @@ int main(void) {
     free(password);
     free(password_confirmation);
 
-    return EXIT_SUCCESS;
+    return ret;
+}
+
+int main(int argc, char **argv) {
+    switch (argc) {
+    case 1:
+        return main_set_password();
+    case 2:
+        if (strcmp(argv[1], "-l") == 0) {
+            return main_remove_password();
+        }
+        // otherwise, fall through
+    default:
+        fprintf(stderr, "Supported options are no options or -l\n");
```

Comment:
I think help message can be improved here, maybe something like:

```
fprintf(stderr, "Run %s without args to set a password, or with -l to remove the password\n", argv[0]);
```

Reply:
Good call